src/doc/yaml/enrollment.yaml

```diff
@@ -0,0 +1,652 @@
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    patch:
+      tags:
+        - Enrollment
+      summary: Update partially one enrollment
+      description: Update partially an enrollment. Payload content is a partial enrollment object compliant with RFC7396.
+      operationId: partialUpdateEnrollment
+      parameters:
+        - name: enrollmentId
+          in: path
+          description: the id of the enrollment
+          required: true
+          schema:
+            type: string
+        - name: finalize
+          in: query
+          description: Flag to indicate that data was collected
+          required: true
+          schema:
+            type: boolean
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Enrollment'
+      responses:
+        '204':
+          description: Update successful
+        '400':
+          description: Bad request
+        '403':
+          description: Update not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+                
+  /v1/enrollment/{enrollmentId}/finalize:
+    put:
+      tags:
+        - Enrollment
+      summary: Finalize one enrollment
+      operationId: finalizeEnrollment
+      parameters:
+        - name: enrollmentId
+          in: path
+          description: the id of the enrollment
+          required: true
+          schema:
+            type: string
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      responses:
+        '204':
+          description: Update successful
+        '400':
+          description: Bad request
+        '403':
+          description: Update not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+                
+
+  /v1/enrollments:
+    post:
+      tags:
+        - Enrollment
+      summary: Retrieve a list of enrollments which match passed in search criteria
+      operationId: findEnrollments
+      parameters:
+        - name: expressions
+          in: query
+          description: The expressions to evaluate.
+          required: true
+          schema:
+            $ref: '#/components/schemas/Expressions'
```

Comment:
Expressions are both in query and in body. I think this 'query' parameter should be removed.